mm2src/coins/tendermint/tendermint_token.rs

```diff
@@ -538,6 +538,16 @@ impl MarketCoinOps for TendermintToken {
         self.platform_coin.send_raw_tx_bytes(tx)
     }
 
+    #[inline(always)]
+    fn sign_raw_tx(&self, _args: &SignRawTransactionRequest) -> SignRawTransactionFut {
+        let ticker = self.ticker.clone();
+        Box::new(
+            async move { Err(RawTransactionError::NotImplemented { coin: ticker }.into()) }
+                .boxed()
+                .compat(),
+        )
+    }
+
```

Comment:
Also, I wonder why we didn't use `unimplemented!()` here

Reply:
Do we want to panic if sign_raw_transaction is called for an unsupported coin?

Reply:
Ok ok make sense...I asked because I saw the usage somewhere else